harfbuzz-sys/Cargo.toml

```diff
@@ -27,8 +27,8 @@ pkg-config = { version = "0.3", optional = true }
 cc = { version = "1", optional = true }
 
 [target.'cfg(target_vendor = "apple")'.dependencies]
-core-graphics = "0.17"
-core-text = "13"
+core-graphics = ">=0.19"
+core-text = ">=15"
```

Comment:
No need for the >=s. This can be `"0.19"` and `"15"` with the same effect.

Reply:
Done.